package-lock.json

```diff
@@ -1,9 +1,24 @@
 {
   "name": "fluid-publish",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
We should use this opportunity to axe this lockfile - which in any case has a version which doesn't agree with the package.json version